test_runner/src/main/kotlin/ftl/run/model/AndroidTestContext.kt

```diff
@@ -15,6 +15,7 @@ data class InstrumentationTestContext(
     val ignoredTestCases: IgnoredTestCases = emptyList(),
     val environmentVariables: Map<String, String> = emptyMap(),
     val testTargetsForShard: ShardChunks = emptyList(),
+    val parameterizedTestsOption: String = "",
```

Comment:
`parameterizedTests` can be acquired from `args`, I think we do not need to have id duplicated here.

Reply:
Will check. It might be left over here.